github-pullrequest-plugin/src/test/java/org/jenkinsci/plugins/github_integration/its/WorkflowITest.java

```diff
@@ -0,0 +1,33 @@
+package org.jenkinsci.plugins.github_integration.its;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.jenkinsci.plugins.github.pullrequest.util.TestUtil.classpath;
+
+
+/**
+ * @author Kanstantsin Shautsou
+ */
+public class WorkflowITest extends AbstractPRTest {
+
+    @Ignore(value = "Fails with jquery")
```

Comment:
@jglick @abayer  why workflow-job doesn't work when it added to test scope? Everything works except jquery that is missing. And jquery plugins releases (where releases?) and dependencies looks *very bad*.